sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/XmlHighlightingTest.java

```diff
@@ -113,12 +113,12 @@ void testHighlightTagWithDoubleQuoteAttribute() throws Exception {
 
   @Test
   void testHighlightMultilineTagWithAttributes() throws Exception {
-    highlight(
-      "<tag att1='value1' \n"
-        + " att2\n"
-        + " = 'value2' att3=\n"
-        + "'value3' att4='multiline \n"
-        + " \" attribute'> </tag>");
+    highlight("""
+      <tag att1='value1'\s
+       att2
+       = 'value2' att3=
+      'value3' att4='multiline\s
```

Comment:
I assume the `\s` is to prevent the space to be trimmed from the string?

Reply:
Yes, all string to textbox conversions were done unsing my IDE, not me. And IDE seems to preserve all spaces.
Aboud `\s` I found [this doc](https://docs.oracle.com/en/java/javase/15/text-blocks/index.html#new-escape-sequences).